sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-files/e-cert-file-footer.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from "@nestjs/common";
+import { FixedFormatFileLine } from "../../../services/ssh/sftp-integration-base.models";
+import { RecordTypeCodes } from "../models/e-cert-integration-model";
+
+/**
+ * Footer of an E-Cert file.
+ * The documentation about it is available on the document
+ * 'CSLP-AppendixF2AsReviewed2016-FileLayouts BC Files V3(HAJ-CB EDITS) In ESDC Folder'.
+ */
+@Injectable()
+export abstract class ECertFileFooter implements FixedFormatFileLine {
+  recordTypeCode: RecordTypeCodes;
```

Comment:
I can see that these 3 variables are declared in the base class and the inherited classes, can they be declared only at the base class?